device-types/Cisco/WS-C2960-48TT-S.yaml

```diff
@@ -0,0 +1,115 @@
+---
+manufacturer: Cisco
+model: Catalyst 2960-48TT-S
+slug: cisco-ws-c2960-48tt-s
+part_number: WS-C2960-48TT-S
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)